packages/ecs/src/lib/World.ts

```diff
@@ -10,24 +10,49 @@ export interface WithWorld {
  * @internal
  */
 export default class World {
+	private static worlds: { [id: string]: World } = {};
```

Comment:
I think it's important to be able to get a world by id instead of reference to the instance. And why would you keep the worlds object outside of the class?

Reply:
Hm we don't keep track of instances anywhere else, I'd like to keep it consistent. This could also lead to potential collisions – you could overwrite or access worlds created by other users. I'd argue that it is super easy for you to keep track of your own worlds, so that's not something we need to provide imho.

Reply:
Regarding keeping the worlds object outside the class: You're correct, it does not necessarily make sense.
However, if we don't track worlds, we can remove it anyways.